### DISCUSSION TITLE
Do the Nextcloud System Requirements Apply to Nextcloud AIO?

### DISCUSSION THREAD

**Discussion by Anvil5465** [nextcloud/all-in-one#5183] [Questions] [answered]
With the first point release of Ubuntu server 24.04.1 now out, I was considering upgrading from 22.04, but I noticed in the [Nextcloud documentation](https://docs.nextcloud.com/server/latest/admin_manual/installation/system_requirements.html) that 22.04 is still the recommended version. Does this recommendation also apply to Nextcloud AIO? Or does it not matter much because it's in a docker container?

**Comment by superjamie** [comment, marked as answer]
That's only relevant if you are doing a manual installation of all Nextcloud components yourself.

The AIO containers package everything needed into their own container images. You can run those container images on any Docker-capable host operating system.

For example, there really should not be any problem running NC-AIO on Alpine Linux or any other Linux distro.
